src/pudl/metadata/fields.py

```diff
@@ -2034,16 +2033,6 @@
                 "required": True,
             }
         },
-        "plant_id_eia": {
-            "constraints": {
-                "required": True,
-            }
```

Comment:
Is this being removed from the `epacems` specific resource metadata because it's no longer a required field? Isn't it one of the primary key columns in the `epacems` table? If so then it should remain required shouldn't it? Or I guess it's automatically considered required by virtue of being part of the PK.

If we're not going to fill in the NA values for `gross_load_mw` and `heat_content_mmbtu`, then those fields should also be removed from this section, since they will also no longer be required, and the NA values will violate the current schema.

Reply:
We talked about this in slack, but I'm gonna repeat it here for transparency. `plant_id_eia` has some NA values so we had to remove if from here and also demote it as a primary key in favor of the less accurate but all filled in `plant_id_epa`. The crosswalk table will now have both fields.